### ISSUE TITLE
Unable to load configuration file '/var/www/nZEDb/nzedb/config/config.php'

### ISSUE THREAD

**Issue by MrHappy** [nZEDb/nZEDb#1794]
Just did a git pull and ran: php cli/update_db.php 1.
Now I get 

PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Unable to load configuration file '/var/www/nZEDb/nzedb/config/config.php'. Make sure it has been created and contains correct settings.' in /var/www/nZEDb/nzedb/config/Configure.php:61
Stack trace:
#0 /var/www/nZEDb/nzedb/config/Configure.php(48): nzedb\config\Configure->loadSettings('config', true)
#1 /var/www/nZEDb/nzedb/config/Configure.php(41): nzedb\config\Configure->loadEnvironment('indexer')
#2 /var/www/nZEDb/indexer.php(27): nzedb\config\Configure->__construct('indexer')
#3 /var/www/nZEDb/cli/update_db.php(21): require_once('/var/www/nZEDb/...')
#4 {main}
  thrown in /var/www/nZEDb/nzedb/config/Configure.php on line 61

**Comment by kevinlekiller** [comment]
Hi MrHappy, you can run this to move it:

https://gist.githubusercontent.com/kevinlekiller/233052179689512a5f58/raw/e674efcdad527af1385dae403dc184499c068504/move_config.php

**Comment by MrHappy** [comment]
thanks...! working fine now

**Comment by shadoolaw** [comment]
I just did a git pull and had this same error but didn't know how to fix it so thought starting from scratch would be a good idea. I am unable to complete the initial web setup like before. The news server setup will not continue to the next page unless I disable SSL. It also doesn't create an admin user or save the configuration and the install lock file. I did manage to put those files in manually but without an admin user I couldn't do anything with the site
